frontend/src/App.tsx

```diff
@@ -27,18 +27,18 @@ function AuthedView({ user }: ApiResponse<"get", "/api/auth/me/">) {
       const csrfToken = getCookie("csrftoken");
       const { error } = await apiClient.DELETE("/api/auth/logout/", {
         headers: {
-          "X-CSRFToken": csrfToken
-        }
+          "X-CSRFToken": csrfToken,
+        },
```

Comment:
Please use [biome](https://biomejs.dev/) for formatting, you can install it in VSCode : https://biomejs.dev/reference/vscode/ . 

I will put a pull request for formatting, but you would need to run biome format to format the code.